Insights/URL/URL_TI.yaml

```diff
@@ -0,0 +1,55 @@
+SchemaVersion: '1.0'
+DataTypes:
+- DataType: ThreatIntelligenceIndicator
+Provider: Sentinel
+Type: KQL
+EntitiesFilter: {}
+BaseQuery: | 
+  let UrlThreatIntel=(URL_Url:string){
+  ThreatIntelligenceIndicator 
+  | where isnotempty(Url) or isnotempty( DomainName )
+  | extend match = Url matches regex @'^\\w{2,6}://' 
+  | extend Url = iff(match, Url, strcat('Unknown://', Url))
+  | extend parsedURL=parse_url(Url) 
+  | evaluate bag_unpack(parsedURL) 
+  | where parse_url(URL_Url)['Host'] == coalesce(tostring(parse_url(Url)['Host']),DomainName, Url) 
+  | parse Description with * 'iType: ' MaliciousType'; State:' State'; Org: 'Org'; Source: 'Source_"|;
```

Comment:
This line is broken, not sure exactly what you were trying to accomplish so I cannot comment on a suitable fix.